controls/postgres_spec.rb

```diff
@@ -66,10 +66,37 @@
   impact 1.0
   title 'Postgresql should be running'
   desc 'Postgresql should be running.'
-  describe service(postgres.service) do
-    it { should be_installed }
-    it { should be_running }
-    it { should be_enabled }
+  case os[:name]
+  when 'ubuntu'
+    case os[:release]
+    when '12.04'
+      describe service(postgres.service) do
+        it { should be_installed }
+        it { should be_running }
+        it { should be_enabled }
+      end
+    when '16.04'
+      describe systemd_service(postgres.service) do
```

Comment:
This may be related to https://github.com/chef/inspec/issues/931, similar things are reported to happen with debian